examples/scalability/cases/ClientsRestart/README.md

```diff
@@ -0,0 +1,240 @@
+NSE_NODE=${NODES[0]}
+if [[ "${TEST_REMOTE_CASE}" == "true" ]]; then
+  NSC_NODE=${NODES[1]}
+else
+  NSC_NODE=${NODES[0]}
+fi
+echo NSE_NODE ${NSE_NODE}, NSC_NODE ${NSC_NODE}
+```
+
+Create helper functions:
+```bash
+. ../define_helper_functions.sh
+```
+
+Deploy network services:
+```bash
+generate_netsvc ${TEST_NS_COUNT}
+```
+```bash
+kubectl apply -f netsvcs.yaml
+```
+
+Deploy endpoints:
+```bash
+create_endpoint_patches ${TEST_NSE_COUNT} ${NSE_NODE} endpoints 0
+```
+```bash
+kubectl apply -k ./endpoints
+```
+```bash
+timeout -v --kill-after=10s 3m kubectl -n ${NAMESPACE} wait pod --timeout=3m -l app=nse-kernel --for=condition=ready
+```
+
+Make sure that all endpoints have finished registration:
+```bash
+checkEndpointsStart ${NAMESPACE} endpoints
+```
+```bash
+EVENT_LIST="${EVENT_LIST} ENDPOINTS_0_STARTED"
+EVENT_TIME_ENDPOINTS_0_STARTED="$(date -Iseconds)"
+EVENT_TEXT_ENDPOINTS_0_STARTED="All endpoints started"
+```
+
+Deploy clients:
+```bash
+create_client_patches ${TEST_NSC_COUNT} ${NSC_NODE} clients-0
+```
+```bash
+kubectl apply -k ./clients-0
+```
+```bash
+timeout -v --kill-after=10s 3m kubectl -n ${NAMESPACE} wait pod --timeout=3m -l app=nsc-kernel --for=condition=ready
+```
+
+```bash
+checkClientsSvid ${NAMESPACE}
+```
+```bash
+EVENT_LIST="${EVENT_LIST} CLIENTS_GOT_SVID"
+EVENT_TIME_CLIENTS_GOT_SVID="$(date -Iseconds)"
+EVENT_TEXT_CLIENTS_GOT_SVID="All clients-0 obtained svid"
+```
+
+```bash
+checkConnectionsCount ${NAMESPACE} "10.0" ${TEST_NS_COUNT}
+```
+```bash
+EVENT_LIST="${EVENT_LIST} CONNECTIONS_0_READY"
+EVENT_TIME_CONNECTIONS_0_READY="$(date -Iseconds)"
+EVENT_TEXT_CONNECTIONS_0_READY="Conns 0 ready"
+```
+```bash
+sleep 15
+```
+
+Delete first batch of clients:
+```bash
+EVENT_LIST="${EVENT_LIST} DELETE_CLIENTS_0"
+EVENT_TIME_DELETE_CLIENTS_0="$(date -Iseconds)"
+EVENT_TEXT_DELETE_CLIENTS_0="Delete clients-0..."
+```
+```bash
+kubectl -n ${NAMESPACE} delete -k ./clients-0 --cascade=foreground
+```
+```bash
+EVENT_LIST="${EVENT_LIST} CLIENTS_DELETED_0"
+EVENT_TIME_CLIENTS_DELETED_0="$(date -Iseconds)"
+EVENT_TEXT_CLIENTS_DELETED_0="Clients-0 deleted"
+```
+
+Create second batch of clients:
+```bash
+create_client_patches ${TEST_NSC_COUNT} ${NSC_NODE} clients-1
+```
+```bash
+kubectl apply -k ./clients-1
+```
+```bash
+timeout -v --kill-after=10s 3m kubectl -n ${NAMESPACE} wait pod --timeout=3m -l app=nsc-kernel --for=condition=ready
+```
+
+```bash
+checkClientsSvid ${NAMESPACE}
+```
+```bash
+EVENT_LIST="${EVENT_LIST} CLIENTS_1_GOT_SVID"
+EVENT_TIME_CLIENTS_1_GOT_SVID="$(date -Iseconds)"
+EVENT_TEXT_CLIENTS_1_GOT_SVID="All clients-1 obtained svid"
+```
+
+```bash
+checkConnectionsCount ${NAMESPACE} "10.0" ${TEST_NS_COUNT}
+```
+```bash
+EVENT_LIST="${EVENT_LIST} CONNECTIONS_1_READY"
+EVENT_TIME_CONNECTIONS_1_READY="$(date -Iseconds)"
+EVENT_TEXT_CONNECTIONS_1_READY="Conns 1 ready"
+```
+```bash
+sleep 15
```

Comment:
Why do we need this sleep?

Reply:
When I added sleeps into tests, my main intent was to capture performance after different phases of the test. For example, to make sure, that when we capture that request/heal finishes, it no longer consumes CPU.